ui/pages/login.tsx

```diff
@@ -30,24 +30,22 @@ import {
   toggleWebauthnSkip,
 } from "../util/webauthnAutoLogin";
 
+type AppConfig = {
+  oidc_webauthn_sequencing_enabled?: boolean;
+};
+
 const Login: NextPage = () => {
   const [flow, setFlow] = useState<LoginFlow>();
   const [isSequencedLogin, setSequencedLogin] = useState(false);
-  const [baseURL, setBaseURL] = useState("");
   const isAuthCode = flow?.ui.nodes.find((node) => node.group === "totp");
 
   useEffect(() => {
-    void fetch("/api/v0/app-config")
+    void fetch("../api/v0/app-config")
```

Comment:
question: are you sure this works with any context path the UI could be deployed at?

Reply:
This is a relative path. We use the same approach in other places already. It should always work [tm], because this component is a page and we can make the assumption that '../' relates to the base url.